app_data/vocabularies/resource_types.yaml

```
@@ -0,0 +1,1057 @@
+    marc21_subtype: ""
+  title:
+    en: Lesson
+    sv: Lektion
+    de: Unterrichtseinheit
+    cs: Lekce
+  tags:
+    - depositable
+    - linkable
+- id: software-computationalnotebook
+  icon: code
+  props:
+    csl: software
+    datacite_general: ComputationalNotebook
+    datacite_type: ""
+    openaire_resourceType: "40"
+    openaire_type: software
+    eurepo: info:eu-repo/semantics/other
+    schema.org: https://schema.org/SoftwareSourceCode
+    subtype: software-computationalnotebook
+    type: software
+    marc21_type: software
+    marc21_subtype: computationalnotebook
+  title:
+    en: Computational notebook
+    sv: Computational notebook
+    de: Computational Notebook
+    cs: Výpočetní notebook
+  tags:
+    - depositable
+    - linkable
+- id: other
+  icon: asterisk
+  props:
+    csl: article
+    datacite_general: Other
+    datacite_type: ""
+    openaire_resourceType: "0020"
+    openaire_type: other
+    eurepo: info:eu-repo/semantics/other
+    schema.org: https://schema.org/CreativeWork
+    subtype: ""
+    type: other
+    marc21_type: other
+    marc21_subtype: ""
+  title:
+    en: Other
+    sv: Övrig
+    de: Sonstige
+    cs: Jiné
+  tags:
+    - depositable
+    - linkable
+- id: physicalobject
+  icon: cube
+  props:
+    csl: graphic
+    datacite_general: PhysicalObject
+    datacite_type: ""
+    openaire_resourceType: "0010"
+    openaire_type: other
+    eurepo: info:eu-repo/semantics/other
+    schema.org: https://schema.org/CreativeWork
+    subtype: ""
+    type: physicalobject
+    marc21_type: physicalobject
+    marc21_subtype: ""
+  title:
+    en: Physical object
+    sv: Fysiskt objekt
+    de: Physikalisches Objekt
+    cs: Fyzický objekt
+  tags:
+    - depositable
+    - linkable
+- id: workflow
+  icon: code fork
+  props:
+    csl: software
+    datacite_general: Workflow
+    datacite_type: ""
+    openaire_resourceType: "0020"
+    openaire_type: other
+    eurepo: info:eu-repo/semantics/other
+    schema.org: https://schema.org/CreativeWork
+    subtype: ""
+    type: workflow
+    marc21_type: workflow
+    marc21_subtype: ""
+  title:
+    en: Workflow
+    sv: Workflow
+    de: Workflow
+    cs: Workflow
+  tags:
+    - depositable
+    - linkable
+# -------------------------------------------------- #
+- id: publication-periodicalissue
+  icon: file alternate
+  props:
+    csl: periodical 
+    datacite_general: Text   
+    datacite_type: ""   
+    openaire_resourceType: "0005"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/other
+    schema.org: https://schema.org/Periodical  
+    subtype: publication-periodicalissue 
+    type: publication
+    marc21_type: publication
+    marc21_subtype: periodicalissue
+  title:
+    en: Periodical issue
+    sv: Tidskriftsutgåva
+    de: Zeitschriftenausgabe
+    cs: Číslo periodika
+  tags:
+    - depositable
+    - linkable
+- id: administrative-circular
+  icon: file alternate
+  props:
+    csl: report 
```

```suggestion
    csl: regulation 
```

Might fit better from [the CSL types](https://docs.citationstyles.org/en/stable/specification.html#terminology)